go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/openshift/cluster-image-registry-operator
 
-go 1.22
+go 1.22.0
+
+toolchain go1.22.3
```

Comment:
Was this automatically added? Does everything work fine without line 5?

Reply:
It was added with `go mod tidy && go mod vendor`. I can test by removing that line